src/test/scala/chiselTests/BoringUtilsSpec.scala

```diff
@@ -302,4 +303,170 @@ class BoringUtilsSpec extends ChiselFlatSpec with ChiselRunners with Utils with
     log should include("Calling .toDefinition fully closes Bar, but it is later bored through!")
     log should include("Can only bore into modules that are not fully closed")
   }
+
+  "Downwards tap from parent to child" should "work" in {
+    val chirrtl = circt.stage.ChiselStage.emitCHIRRTL(
+      new RawModule {
+        class Foo() extends RawModule {
+          val internalWire = Wire(Bool())
+        }
+
+        val foo = Module(new Foo())
+        val outProbe = IO(probe.Probe(Bool()))
+        val out = IO(Bool())
+
+        probe.define(outProbe, BoringUtils.tap(foo.internalWire))
+
+        out := BoringUtils.tapAndRead(foo.internalWire)
+      },
+      Array("--full-stacktrace", "--throw-on-first-error")
+    )
+    matchesAndOmits(chirrtl)(
+      "output bore : Probe<UInt<1>>",
+      "output out_bore : Probe<UInt<1>>",
+      "define bore = probe(internalWire)",
+      "define out_bore = probe(internalWire)",
+      "define outProbe = foo.bore",
+      "out <= read(foo.out_bore)"
```

Comment:
I know I said to tighten this up. However, continuing to include the `module` will make this more readable. This may require not using anonymous modules or also using `suggestName` on the anonymous modules to give them names that will show up in the FIRRTL.